### ISSUE TITLE
iso19139-hnap schema rule fails

### ISSUE THREAD

**Issue by RousseauLambertLP** [geopython/pygeometa#92]
I looked at https://github.com/geopython/pygeometa/blob/master/pygeometa/templates/iso19139-hnap/main.j2#L421-454 and saw that only if the CRS value is not equal to '4326' the BBOX value should be written on the same line. If the CRS is set to 4326, it should use the {else} tag where the max lat, lon and min lat lon should be written separately on different lines.

We have a few models which uses CRS 4326 and they all have the same behaviour. 

Ex for GDPS (xml):
```
<gmd:polygon>
  <gml:Polygon gml:id="P001" srsDimension="2" srsName="4326">
    <gml:exterior>
      <gml:LinearRing>
        <gml:posList srsDimension="2" srsName="4326">-180  -90 -180  90  180  90  180  -90 -180  -90</gml:posList>
     </gml:LinearRing>
   </gml:exterior>
</gml:Polygon>
```
In the YAML:
```
spatial:
    datatype: grid
    geomtype: solid
    crs: 4326
    bbox: -180, -90, 180, 90
```

Copying @alexandreleroux and @tomkralidis for comments.

**Comment by RousseauLambertLP** [comment]
I pushed a change in my branch to correct this issue. https://github.com/RousseauLambertLP/pygeometa/commit/bc097917d64473ce8c0bd702ad2b148482076a2b

The problem was that in the YAML we had:
```
spatial:
    datatype: grid
    geomtype: solid
    crs: 4326
```
The value 4326 is read has integer and in the main.j2 files:
```
            {% if record['spatial']['crs'] != '4326' %}
```
The brackets around 4326 passed it as string. Thus, when comparing an integer to a string it failed... I simply removed the ' ' around 4326 in all 3 (iso19139-hnap, iso19139 and wmo-cmp) main.j2 files.

If you are ok with this solution, I will do a pull request.
